spec/support/api/helpers.rb

```diff
@@ -50,6 +50,8 @@ def entity_action_identifier(type, entity, action, method = :post)
         end
 
         def action_identifier(type, action, selection = :resource_actions, method = :post)
+          method = action == :read ? :get : :post
```

Comment:
Does this effectively ignore the passed in `method` then?

Reply:
Oh I misread your comment, so yes it does ignore it.  In that case, let's maybe change the method signature to say:

```diff
-         def action_identifier(type, action, selection = :resource_actions, method = :post)
+         def action_identifier(type, action, selection = :resource_actions, _method = :post)
```

Reply:
yea. I debated fixing all the callers or here.
Would like your thoughts on whether it is better to "do it right" and have a lot of change/harder to back port or do it quick.

Reply:
heh. your second comment just showed up. just pushed that change.